playbooks/nagios/README.md

```diff
@@ -0,0 +1,49 @@
+# Nagios Example Run
+
+The `nagios-target` and `nagios-server` roles can be used to setup a complete Nagios monitoring for any environment. The `nagios-target` role will prepare the targets with the correct monitoring configuratoin for use with the NRPE (Nagios Remote Plugin Executor). The target role will selectively enable the correct monitoring plugins (and correctly configured) per targets.
+
+Below is an example inventory file for setting up the Nagios server and targets. Before executing, ensure that access to the target servers is enabled - i.e.: SSH key login for root. 
+
+Example run of the playbook:
+> ansible-playbook -i \<path_to_inventory\> setup_nagios.yml
+
+
+```
+[all:vars]
+ansible_ssh_user=root
+
+# Infrastructure Server
+[infra]
+dns.infra.example.com nagios_services=dns
+nfs.infra.example.com nagios_services=nfs
+
+[infra:vars]
+hostgroup_name=infra-servers
+hostgroup_alias=Infrastructure Servers
+
+# OpenShift 3 environment
+[openshift]
```

Comment:
@etsauer yes, I totally agree with that, but I have a lot of upcoming changes to this, so I'd like to get this initial submit done and then we can do follow-up PRs to incorporate those sort of things. OK?